#input_clusters_topic: "laser_clustering/clusters" #"laser_clustering/clusters" #"laser_overlap_trace/clusters"
input_clusters_topic: "laser_overlap_trace/clusters"
results4meters_topic: "~results4meters"
results4scans_topic: "~results4scans"
frame_id: "laser_link"
#classifier_file: "SVM_classifier.p" #place the file inside the classification_files folder.
#classifier_file: "LDA_classifier.p" #place the file inside the classification_files folder.
classifier_file: "GaussianNB_classifier.p"
publish_viz: True
viz_topic: "~viz_req"
dt: 25
human_speed: 5
distance: 4
min_distance: 1
timewindow: 40
write_to_file: True
file: '/home/ros/Desktop/statNB.csv' #'/media/sf_new_shared/fhag_bags/hprResults/stat_10'
pca_file: "PCA_object.p"  #place the file inside the classification_files folder.
